server/src/config.ts

```diff
@@ -45,14 +47,24 @@ export const ZMQ_DEV_PORT = getConfigField('ZMQ_DEV_PORT', '8002');
 export const ZMQ_HOST = getConfigField('ZMQ_HOST', '127.0.0.1');
 export const HASTIC_API_KEY = getConfigField('HASTIC_API_KEY');
 export const GRAFANA_URL = normalizeUrl(getConfigField('GRAFANA_URL', null));
+
 // TODO: save orgId in analytic_units.db
 export const ORG_ID = getConfigField('ORG_ID', 1);
+
+export enum AlertTypes {
+  webhook = 'webhook',
+  alertmanager = 'alertmanager'
+};
+export const HASTIC_ALERT_TYPE = getConfigField('HASTIC_ALERT_TYPE', AlertTypes.webhook, _.keys(AlertTypes));
```

Comment:
if you make keys UPPER_CASE:
```suggestion
export const HASTIC_ALERT_TYPE = getConfigField('HASTIC_ALERT_TYPE', AlertTypes.webhook, _.values(AlertTypes));
```